docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Jakarta_Server_Faces.adoc

```diff
@@ -0,0 +1,166 @@
+:module-url: https://raw.githubusercontent.com/wildfly/wildfly/main/jsf/multi-jsf-installer/src/main/resources
+
+[[Jakarta_Server_Faces]]
+= Jakarta Server Faces Configuration
+
+[abstract]
+
+Jakarta Server Faces configuration is handled by the Jakarta Server Faces subsystem. The Jakarta Server Faces subsystem
+allows multiple Jakarta Server Faces implementations to be installed on the same WildFly server. In particular, any
+version that implements spec level 4.0 or higher can be installed. For each Jakarta Server Faces
+implementation, a new slot needs to be created under `jakarta.faces.impl`, `jakarta.faces.api`, and
+`org.jboss.as.jsf-injection`. When the Jakarta Server Faces subsystem starts up, it scans the module path to find all
+the Jakarta Server Faces implementations that have been installed. The default Jakarta Server Faces implementation that
+WildFly should use is defined by the `default-jsf-impl-slot` subsystem attribute.
+
+[[installing-a-new-jsf-implementation-via-feature-pack]]
+== Installing a new Jakarta Server Faces implementation via a feature pack
+
+WildFly supports provisioning a server using the Galleon tool, which allows an administrator to provision a server with
+only the desired features, which are delivered as feature packs. For more information, see link:Galleon_Guide{outfilesuffix}[Provisioning WildFly with Galleon].
+For an example of such a feature pack, see the https://github.com/wildfly-extras/wildfly-myfaces-feature-pack[WildFly MyFaces Feature Pack]
+project in the WildFly Extras GitHub organization.
+
+As a quick start, to provision a server using this feature pack, one might use a commandline like the following:
+
+[source,bash]
+-----
+$ galleon.sh provision myfaces_server.xml --dir=$SERVER_DIR
+-----
+
+.myfaces_server.xml
+[source,xml]
+-----
+<?xml version="1.0" ?>
+<installation xmlns="urn:jboss:galleon:provisioning:3.0">
+  <feature-pack location="org.wildfly:wildfly-galleon-pack:{wildfly.version}">
+    <default-configs inherit="true"/>
+    <packages inherit="true"/>
+  </feature-pack>
+  <feature-pack location="org.wildfly:wildfly-myfaces-feature-pack:{feature-pack.version}">
+    <default-configs inherit="true"/>
+    <packages inherit="true"/>
+  </feature-pack>
+  <config model="standalone" name="standalone.xml">
+    <layers>
+      <!-- Base layer -->
+      <include name="management"/>
+      <include name="myfaces"/>
+    </layers>
+  </config>
+  <options>
+    <option name="optional-packages" value="passive+"/>
+    <option name="jboss-fork-embedded" value="true"/>
+  </options>
+</installation>
+-----
+
+[[manually-installing-a-new-jsf-implementation]]
+== Manually installing a new Jakarta Server Faces implementation
```

Comment:
I feel a bit doubtful about documenting this. I don't think we actually want anyone to do this.

Reply:
> I feel a bit doubtful about documenting this. I don't think we actually want anyone to do this.

I think I agree. IIRC, it's re-addition was explicitly requested, but the feature pack changes things a bit. If someone wants to do this manually, the FP should show how to do that well enough that we need not document that here.